packages/core/test/integration/data-types/data-types.test.ts

```diff
@@ -1333,9 +1361,20 @@ describe('DataTypes', () => {
       }
     });
 
-    it(`is deserialized as a string when DataType is not specified`, async () => {
-      await testSimpleInOutRaw(vars.User, 'dateAttr', '2022-01-01', '2022-01-01');
-    });
+    if (dialect.name === 'oracle') {
+      it(`is deserialized as a date when DataType is not specified`, async () => {
```

Comment:
Why is this not deserialized as a string? We prefer to have uniformity whenever possible